cypress/e2e/settings/remote-storage/remote-storage-callout.cy.js

```diff
@@ -0,0 +1,73 @@
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import { Configurations } from '../../../support/fragments/settings/remote-storage';
+import Users from '../../../support/fragments/users/users';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import TopMenu from '../../../support/fragments/topMenu';
```

Comment:
'TopMenu' is defined but never used. Allowed unused vars must match /React/u

Reply:
removed